### PR TITLE
[FLINK-19527][Doc] Common errors when running with docker on windows

### PR DESCRIPTION
**Pull request by haseeb1431**
## What is the purpose of the change

Try flink experiments spins up couple of docker containers which works fine with non-windows containers. In some cases, when user is running docker on windows, some of script wouldn't work so we have updated the documentation with a workaround. 

Following is the preview

image

### CODE REVIEW

File: docs/try-flink/table_api.md [hunk 1]
```diff
@@ -44,6 +44,13 @@ It also assumes that you are familiar with basic relational concepts such as `SE
 If you get stuck, check out the [community support resources](https://flink.apache.org/community.html).
 In particular, Apache Flink's [user mailing list](https://flink.apache.org/community.html#mailing-lists) consistently ranks as one of the most active of any Apache project and a great way to get help quickly. 
 
+
+### Docker on windows
```
Comment (sjwiesman) [inline docs/try-flink/table_api.md:48]: We capitalize all proper words in titles along with a new line between the header and the first 

```suggestion
### Docker on Windows

```

File: docs/try-flink/table_api.md [hunk 2]
```diff
@@ -44,6 +44,13 @@ It also assumes that you are familiar with basic relational concepts such as `SE
...
+if you're running docker on windows and your data generator container is failing to start then please ensure that you're using right shell.
```
Comment (sjwiesman) [inline docs/try-flink/table_api.md:49]:
```suggestion
If you're running docker on windows and your data generator container is failing to start then please ensure that you're using the right shell.
```

Grammar

File: docs/try-flink/table_api.md [hunk 3]
```diff
@@ -44,6 +44,13 @@ It also assumes that you are familiar with basic relational concepts such as `SE
...
+For example `docker-entrypoint.sh` for `table-walkthrough_data-generator_1` container requires bash shell.
```
Comment (sjwiesman) [inline docs/try-flink/table_api.md:50]:
```suggestion
For example, `docker-entrypoint.sh` for `table-walkthrough_data-generator_1` container requires bash. 
```

File: docs/try-flink/table_api.md [hunk 4]
```diff
@@ -44,6 +44,13 @@ It also assumes that you are familiar with basic relational concepts such as `SE
 If you get stuck, check out the [community support resources](https://flink.apache.org/community.html).
 In particular, Apache Flink's [user mailing list](https://flink.apache.org/community.html#mailing-lists) consistently ranks as one of the most active of any Apache project and a great way to get help quickly. 
 
+
+### Docker on windows
+if you're running docker on windows and your data generator container is failing to start then please ensure that you're using right shell.
+For example `docker-entrypoint.sh` for `table-walkthrough_data-generator_1` container requires bash shell. 
+If you don't have it, it will throw an error `standard_init_linux.go:211: exec user process caused "no such file or directory"`. 
+So a workaround is to swith shell to `sh` from `bash` on the first row in `docker-entrypoint.sh` to make it work.
```
Comment (sjwiesman) [inline docs/try-flink/table_api.md:52]:
```suggestion
A workaround is to switch shell to `sh` from `bash` on the first row in `docker-entrypoint.sh`.
```

**Review by sjwiesman — CHANGES_REQUESTED** [review]
A few comments on the grammar. Two other points: 

1) We put the Jira and component in the commit. Your PR title is correct and should be the commit message. 

2) Take a look at the docs contributing guidelines. I'd like to wrap this in a `note` annotation. 

https://flink.apache.org/contributing/docs-style.html#annotations

**Comment by haseeb1431** [comment]
@sjwiesman Thanks for the inputs. I have addressed all the requested changes. Regarding the wrapping, I have wrapped the message in the "info" because there is no note annotation unless I am missing something. Thanks

image

**Comment by sjwiesman** [comment]
That's what I meant :) Sorry for being confusing. 

LGTM, merging . . .

**Pull request closed without merging**